pinot-core/src/test/java/com/linkedin/pinot/core/startree/TestStarTreeIntegrationTest.java

```diff
@@ -182,13 +223,18 @@ private void buildSegment(String segmentDirName, String segmentName)
         map.put(dimName, dimName + "-v" + row % (NUM_DIMENSIONS - i));
       }
 
-      // Time column.
-      map.put("daysSinceEpoch", row % 7);
+      long seed = System.nanoTime();
+      LOGGER.info("Random seed for metric values: {}", seed);
```

Comment:
Using assert to print such messages seems odd to me. We should fix the logging level in that case.